inc/class-storefront.php

```diff
@@ -155,7 +155,7 @@ public function widgets_init() {
         $widget_tags = array(
           'before_widget' => '<div id="%1$s" class="widget %2$s">',
           'after_widget'  => '</div>',
-          'before_title'  => '<span class="widget-title">',
+          'before_title'  => '<span class="h3 widget-title">',
```

Comment:
I thought we were using `h5`s for widget titles? You can use `.gamma` class for h3 styling.